CHANGES/4749.misc.rst

```diff
@@ -0,0 +1,2 @@
+Session HTTP methods now have full request typing signatures.
```

Comment:
That trailing period is probably unnecessary either here or after the byline. See the PR docs preview for how it's rendered.
```suggestion
Session HTTP methods now have full request typing signatures
```

Reply:
Fixed. https://github.com/aio-libs/aiohttp/pull/8463/commits/c111e2e3acc6490f9f0071d4181f4ddcb5348fff

Reply:
Marking as resolved.